### DISCUSSION TITLE
Training on grayscale has poor performance on color

### DISCUSSION THREAD

**Discussion by daskandalis** [talmolab/sleap#1715] [Help!]
I have very unequal performance on the same video when it's in grayscale or in color (trained in grayscale, worse in color). I have two copies of the same video, one loaded in grayscale and the other in color. The model training_config.json suggests grayscale will be enforced:

```
"preprocessing": {
            "ensure_rgb": false,
            "ensure_grayscale": true,
            ...
        }
```
Is this expected? Is there an option to force the conversion to grayscale during inference?

**Comment by daskandalis** [comment]
Toggling grayscale in the sleap-label interface is sufficient. When the videos are on grayscale mode, they're tracked well, when they're in color mode, the tracking is poor.

**Reply by Marielamend** [reply]
Hello @daskandalis,

Yes, selecting the toggle feature under the videos tab converts a video/all videos in a project into grayscale if not already. 
![Screenshot 2024-03-20 133810](https://github.com/talmolab/sleap/assets/120222245/20a660be-8fe5-4b78-a674-7eb377d024b3)

There is also a way of converting it under Input Data options in the training pipeline, but this seems to not convert all videos in a project into grayscale in some cases. I can put this under an issue, and our team can further look into it. Thank you for your inquiry! 
![Screenshot 2024-03-20 133157](https://github.com/talmolab/sleap/assets/120222245/2947bf13-8a5b-43ba-b87b-dc5383aa25f5)

Best, 
Mariela
